hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -0,0 +1,256 @@
+                            <TableCell align="right">{row.qty}</TableCell>
+                            <TableCell align="right">{row.time}</TableCell>
+                            <TableCell align="left">{row.condition}</TableCell>
+                            <TableCell className={styles.widthBuffer} />
+                        </TableRow>
+                    ))}
+                </TableBody>
+            );
+
+        case "Order pending":
+            return (
+                <TableBody>
+                    {items.map((row) => (
+                        <TableRow key={row.name}>
+                            <TableCell align="left">
+                                <img
+                                    className={styles.itemImg}
+                                    src={row.url}
+                                    alt={row.name}
+                                />
+                            </TableCell>
+                            <TableCell align="left">{row.name}</TableCell>
+                            <TableCell align="right">{row.reqQty}</TableCell>
+                            <TableCell align="right">{row.grantQty}</TableCell>
+                            <TableCell className={styles.widthBuffer} />
+                        </TableRow>
+                    ))}
+                </TableBody>
+            );
+        default:
+            return null;
+    }
+};
+const EmptyTable = ({ title }) => {
+    switch (title) {
+        case "Checked out items":
+            return (
+                <Paper elevation={3} className={styles.empty} square={true}>
+                    You have no items checked out yet. View our inventory.
+                </Paper>
+            );
+        case "Returned items":
+            return (
+                <Paper elevation={3} className={styles.empty} square={true}>
+                    Please bring items to the tech table and a tech team member will
+                    assist you.
+                </Paper>
+            );
+        default:
+            return null;
+    }
+};
+
+const ChipStatus = ({ status }) => {
+    switch (status) {
+        case "ready":
+            return (
+                <Chip
+                    icon={<CheckCircle />}
+                    label="Ready for pickup"
+                    className={styles.chipGreen}
+                />
+            );
+        case "pending":
+            return (
+                <Chip
+                    icon={<WatchLater />}
+                    label="In progress"
+                    className={styles.chipOrange}
+                />
+            );
+        case "error":
+            return (
+                <Chip
+                    icon={<Error />}
+                    label="Please visit the tech station"
+                    className={styles.chipRed}
+                />
+            );
+        default:
+            return null;
+    }
+};
+
+export const ItemTable = ({ items, title, status }) => (
```

Comment:
Split this out into 3 separate components (exported so you can use a named imports) for each of the 3 tables. That way you can eliminate most of the switch statement functions above, and not rely on hard-coded strings that are subject to change.